src/navigation/NavigationTypes.ts

```diff
@@ -1,13 +1,105 @@
-export interface ProfessionParamList {
+import { DocumentsType, DocumentType } from '../constants/endpoints'
+
+export interface DocumentResultType extends DocumentType {
+  result: 'correct' | 'incorrect' | 'similar'
```

Comment:
Please use the RESULT_TYPE from data.ts here. Or better create a type for this. Would be nice to refactor so not everywhere in the code the there are string comparisons.

I think the best way would be, as 'normal' enums have some disadvantages.
```
    const Result = {
      correct: 0,
      incorrect: 0,
      almostCorrect: 0
    } as const;
    type ResultsType = typeof Result[keyof typeof Result];
    const myResult: ResultsType = Result.correct
    if(myResult === Result.incorrect) {
      ...
    }
```

What do you think? But anyway this should be done in a separate ticket is guess.

Reply:
I did some more refactoring in my other PRs in regards to this result stuff, could you check whether that is enough already? Otherwise I would definitely do it in another PR/ticket as this was not part of the ticket and I would merge this ASAP.